packages/agents/sdk/test/sdkUtils.spec.ts

```diff
@@ -65,6 +66,14 @@ describe("SdkUtils", () => {
     });
   });
 
+  describe("#parseConnextTransactionReceipt", () => {
+    it("happy: should work", async () => {
+      const res = nxtpUtils.parseConnextTransactionReceipt(mock.ethers.receipt());
```

Comment:
Fix this test by using a real xcall receipt.

Reply:
yeah good idea, will add it.